PostProcessing/Runtime/Effects/DepthOfField.cs

```diff
@@ -134,7 +133,17 @@ public override void Render(PostProcessRenderContext context)
             // Material setup
             float scaledFilmHeight = k_FilmHeight * (context.height / 1080f);
             var f = settings.focalLength.value / 1000f;
-            var s1 = Mathf.Max(settings.focusDistance.value, f);
+            var focusDistance = settings.focusDistance.value;
+#if UNITY_2018_2_OR_NEWER
+            if (context.physicalCamera)
+            {
+                f = context.camera.focalLength / 1000f;
+                scaledFilmHeight = (context.camera.sensorSize.y/1000f) * (context.height / 1080f);
+            }
+            if (context.cameraHasInterestPosition)
+                    focusDistance = Vector3.Distance(context.cameraInterestPosition,context.camera.gameObject.transform.position);
+#endif
```

Comment:
For the whole block (134-145), if physcam is enabled:
`context.height / 1080f` is done twice.
`f` and `scaledFilmHeight` are computed twice.